content/docs/started/workstation/getting-started-linux.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Kubeflow on Linux"
+description = "Install Kubeflow on Linux"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+  more quickly
+-->
+
+## Introduction
+
+For Linux systems you have options for servers and desktops. The server options apply to
+the desktop as well.
+
+## Linux Server
+
+For linux servers you can install Kubeflow natively. This is perfect for
+linux hosts and virtual machines, such as VMs in OpenStack, VMware or public clouds like
+GCP, AWS and Azure.
+
+### MicroK8s
+
+[MicroK8s](https://microk8s.io) runs natively on most Linux distributions. It requires
+[snapd](https://snapcraft.io/docs/getting-started), which is pre-installed on Ubuntu
+and Ubuntu derivative operating systems.
+
+1. Install MicroK8s - `snap install microk8s --classic`
+2. Follow the getting started guid for Kubeflow on an
+[existing Kubernetes cluster](/docs/started/k8s/overview/).
+
+## Linux Desktop
+
+### Kubeflow Appliance
+
+A Kubeflow appliance is a virtual machine that has Kubeflow already installed. Once the
+necessary supporting software is installed no further installation steps are required.
+
+#### MiniKF
+
+MiniKF is a predefined virtual machine that installs onto VirtualBox through Vagrant.
+The only following applications are required to use MiniKF:
+
+- Install [Vagrant](https://www.vagrantup.com/downloads.html)
+- Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+
+The full set of instructions are available on the
+[MiniKF getting started](/docs/started/getting-started-minikf/) page.
+
+### Linux Appliance
+
+A Linux appliance is a virtual machine that holds the linux operating system. From there
+you have complete choice over Kubernetes and Kubeflow, which offers the greatest degree
+of flexibility. You only need to install a single application to follow this path:
+
+- Install [Multipass](https://multipass.run/#install)
+
+The instructions on [Multipass and MicroK8s getting started](/docs/started/getting-started-multipass/)
```

Comment:
This link is broken. Please check all links in this PR. I'm stopping with that particular check here. :)